src/main/java/com.gigssandbox/UserMessenger.java

```diff
@@ -0,0 +1,79 @@
+package com.gigssandbox;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Collection;
+import java.util.Properties;
+import java.util.Scanner;
+
+class UserMessenger {
+
+    private static final Scanner in = new Scanner(System.in);
+    private Properties properties = new Properties();
+
+    UserMessenger() {
+        try {
+            properties.load(new FileInputStream("src/main/resources/strings.properties"));
+        } catch (IOException e) {
+            sendMessage("properties_error");
+            System.exit(1);
+        }
+    }
+
+    String readString() {
+        String input = "";
+        while (input.isEmpty()) {
+            input = in.nextLine();
+        }
+        return input;
+    }
+
+    int readInt() {
+        return in.nextInt();
+    }
+
+    void writeStringToConsole(String s) {
+        System.out.println(s);
+    }
+
+    void sendMessage(String propertyName) {
+        System.out.println(getStringFromProperties(propertyName));
+    }
+
+    void sayGoodbye() {
+        sendMessage("say_goodbye");
+        System.exit(0);
```

Comment:
Why System.exit()? 

May be right now it does not matter but in future you are going to have additional logic (closing sockets, updating database after user left and etc) and putting System.exit() may imposes restriction on graceful shutdown design

Reply:
purpose of this method is to close app, so in this situation its name is incorrect

Reply:
but it is going to finish without exiting

Reply:
When this method is called, there is "Process finished with exit code 0" in the console, and the program stops running